src/m365/spo/commands/list/list-remove.ts

```diff
@@ -41,7 +42,8 @@ class SpoListRemoveCommand extends SpoCommand {
       Object.assign(this.telemetryProperties, {
         id: (!(!args.options.id)).toString(),
         title: (!(!args.options.title)).toString(),
-        force: (!(!args.options.force)).toString()
+        force: (!(!args.options.force)).toString(),
+        recycle: (!(!args.options.recycle)).toString()
```

Comment:
Actually, this is an incorrect way of logging telemetry. A better example can be found here: https://github.com/pnp/cli-microsoft365/blob/d92b1c20cbc47e4fc80de6d1bde11792ecebe67c/src/m365/flow/commands/owner/owner-remove.ts#L45-L56

Let's fix that for the existing options as well.

Reply:
Updated